pkgs/servers/pulseaudio/default.nix

```
@@ -144,6 +151,11 @@ stdenv.mkDerivation rec {
     wrapProgram $out/libexec/pulse/gsettings-helper \
      --prefix XDG_DATA_DIRS : "$out/share/gsettings-schemas/${pname}-${version}" \
      --prefix GIO_EXTRA_MODULES : "${lib.getLib dconf}/lib/gio/modules"
+  '' + lib.optionalString (stdenv.isDarwin) ''
+    for file in $out/lib/pulse*/modules/*.dylib; do
+      ln -s "''$file" "''${file%.dylib}.so"
+      ln -s "''$file" "$out/lib/pulseaudio/`basename ''$file .dylib`.so"
```

```suggestion
      ln -s "''$file" "$out/lib/pulseaudio/$(basename ''$file .dylib).so"
```
Please add a comment why we want to do this.

---

Done.